onnxruntime/contrib_ops/webgpu/bert/flash_attention.cc

```diff
@@ -143,7 +143,7 @@ Status FlashAttentionProgram::GenerateShaderCode(ShaderHelper& shader) const {
   const max_k_step: u32 = 16u;
   const vec_factor: u32 = 4u;
   const qkv_head_size_vec: u32 = qkv_head_size / vec_factor;
-  const min_value = f32(-3.402823e+38f);;
+  const min_value : q_element_t = q_element_t(-65504.0);
```

Comment:
is q_element_t possible to be f32? if so, should we use -3.402823e+38 as min_value in this case